authors/apps/articles/renderers.py

```diff
@@ -1,7 +1,36 @@
 import json
+from rest_framework.renderers import JSONRenderer
 
 from authors.apps.authentication.default_renderer import BaseRenderer
 
 
 class ArticleJSONRenderer(BaseRenderer):
     data = 'article'
+
+
+class ArticleShareLinkRenderer(JSONRenderer):
+    # Returns social media share links
+
+    charset = 'utf-8'
+
+    @classmethod
+    def render(self, data, media_type=None, renderer_context=None):
+        # present a link that is in string format
+
+        return json.dumps({
+            'link': data
+        })
+
+
+class ArticleEmailLinkRenderer(JSONRenderer):
+    # Returns social media share links
+
+    charset = 'utf-8'
+
+    @classmethod
```

Comment:
@frankopkusianwar, could you use the `Base renderer` to implement your json rendering